src/main/java/io/confluent/connect/jdbc/dialect/Db2DatabaseDialect.java

```diff
@@ -161,4 +161,11 @@ public String buildUpsertQueryStatement(
     builder.append(")");
     return builder.toString();
   }
+
+  @Override
+  protected String sanitizedUrl(String url) {
+    // DB2 has semicolon delimited property name-value pairs
+    return super.sanitizedUrl(url)
+                .replaceAll("(?i)(;password=)[^;]*", "$1****");
```

Comment:
Maybe `"(?i)([:;]password=)[^;]*"` in case they put the password first (right after the colon) like `"jdbc:db2://sysmvs1.stl.ibm.com:5021/STLEC1:password=dbadm;user=dbadm;"`